### ISSUE TITLE
v1.7.0 SET Invalid Grammar

### ISSUE THREAD

**Issue by rayliverified** [xnuinside/simple-ddl-parser#288]
**Describe the bug**
SET grammar doesn't account for `CHARACTER SET` and breaks when encountering that combination.

**To Reproduce**
```sql
CREATE TABLE `table_notes` (
  `id` int NOT NULL AUTO_INCREMENT,
  `notes` varchar(255) CHARACTER SET utf8mb3 COLLATE utf8mb3_general_ci NOT NULL,
);
```

`set('A', 'B', 'C')` works. Set defaults also works.

**Comment by xnuinside** [comment]
*facepalm* I will fix it today in 1.7.1 and will add test case, thank you!

**Comment by xnuinside** [comment]
@rayliverified didn't have time in previous days, sorry for that, fix released in 1.7.1, test added https://github.com/xnuinside/simple-ddl-parser/pull/289

**Comment by rayliverified** [comment]
Thank you 🥇
